cli/azd/pkg/environment/storage_blob_data_store.go

```diff
@@ -143,6 +153,22 @@ func (sbd *StorageBlobDataStore) Save(ctx context.Context, env *Environment, opt
 		return fmt.Errorf("uploading .env: %w", describeError(err))
 	}
 
+	// Upload Infra Files if any
+	filesToUpload := sbd.azdContext.GetEnvironmentInfraFiles(env.name)
+
+	for _, file := range filesToUpload {
+		fileName := filepath.Base(file)
+		fileBuffer, err := os.Open(file)
+		if err != nil {
+			return fmt.Errorf("failed to open file for upload: %w", err)
+		}
+		defer fileBuffer.Close()
+		err = sbd.blobClient.Upload(ctx, fmt.Sprintf("%s/%s/%s", env.name, InfraFilesDir, fileName), fileBuffer)
+		if err != nil {
+			return fmt.Errorf("uploading infra file: %w", err)
+		}
+	}
```

Comment:
Instead of being selective here - what if we just upload any files that are in the environment folder? That would solve this specific use case as well as any other similar issues.

Reply:
@wbreza I wanted to be more specific in case users have local tmp, or test files in the local environment folder that was not meant to be persisted and to follow the pattern of the environment items being defined ( configfile, env file) by adding infrafiles